src/os/kstore/KStore.cc

```diff
@@ -18,9 +18,9 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <unistd.h>
-#if defined(__FreeBSD__)
-#include <sys/param.h>
-#include <sys/mount.h>
+#if defined(__FreeBSD__) || defined(__APPLE__)
+#  include <sys/param.h>
```

Comment:
the blanks here are debatable...

Reply:
I believe I saw this approach all over the boost library, and I liked how it visually helps to see the includes which are conditional. Otherwise, it's kind of lost in a plain `#<directive>` list

Reply:
Personally I like this style and I wouldn't have an issue with accepting till our coding standards doc says against.